### PR TITLE
New participant input fields

### PR DESCRIPTION
**Pull request by dmjohnspor**

### CODE REVIEW

File: frontend/src/components/ParticipantTableComponent/PrevPointTableBody.js [hunk 1]
```diff
@@ -47,6 +47,9 @@ const PrevPointTableBody = props => {
             <TableCell aria-label="tcell" className={classes.mobileVisibility}>
               <PrevPointCopy>{participant.date_of_birth}</PrevPointCopy>
             </TableCell>
+            <TableCell aria-label="tcell">
```
Comment (MikeyManoguerra) [inline frontend/src/components/ParticipantTableComponent/PrevPointTableBody.js:50]: Can you add the mobile visibility class here? and also move it to left in the table, lets put it after last name

File: frontend/src/components/ParticipantForm.js [hunk 2]
```diff
@@ -96,6 +96,32 @@ const ParticipantForm = ({
           />
         </FormControl>
       </Grid>
+      <Grid item xs={12} sm={6}>
```
Comment (MikeyManoguerra) [inline frontend/src/components/ParticipantForm.js:99]: This was not in the ticket, but while reviewing this, the inputs get minuscule before switching to the mobile view when the sidebar is open.

can you remove the `sm={6}` on each input that has it , and replace it with `md={6}`?
can you also do the same to the `VisitForm.js`? Don't forget to update the snapshot!

File: frontend/src/components/ParticipantForm.js [hunk 3]
```diff
@@ -96,6 +96,32 @@ const ParticipantForm = ({
...
+        <FormControl disabled={isDisabled}>
+          <InputLabel htmlFor="sep_id">SEP id</InputLabel>
```
Comment (MikeyManoguerra) [inline frontend/src/components/ParticipantForm.js:101]: capitalize to be SEP ID

**Review by MikeyManoguerra — CHANGES_REQUESTED** [review]
Looks good, a couple of small requests to address.

File: frontend/src/components/ParticipantTableComponent/PrevPointTableBody.js (hunk 1 above)
Comment (dmjohnspor) [reply frontend/src/components/ParticipantTableComponent/PrevPointTableBody.js]: I addressed the issue.

File: frontend/src/components/ParticipantForm.js (hunk 2 above)
Comment (dmjohnspor) [reply frontend/src/components/ParticipantForm.js]: I addressed the issue.

File: frontend/src/components/ParticipantForm.js (hunk 3 above)
Comment (dmjohnspor) [reply frontend/src/components/ParticipantForm.js]: The change was made.

File: frontend/src/constants/index.js [hunk 4]
```diff
@@ -19,6 +19,7 @@ export const PARTICIPANT_LIST_TABLE_TITLES = [
   { value: true, title: "Last Name" },
   { value: false, title: "Gender" },
   { value: false, title: "DOB" },
+  { value: false, title: "SEP id" },
```
Comment (MikeyManoguerra) [inline frontend/src/constants/index.js:22]: capitalize ID

**Review by MikeyManoguerra — CHANGES_REQUESTED** [review]
Looks good except one minor thing. 
However, I pulled the branch down, and realized the ticket was incomplete. 
can you add `maiden_name` and `sep_id` in the map fn here:

https://github.com/CodeForPhilly/prevention-point/blob/99b5fd3c61f5c6c8b64d1e87f2fab4e33a6e4bdf/frontend/src/components/QueueComponent/QueueTable/QueueTable.js#L74

they are on `x.participant`

The result should be, when you click on the actions button on the left side of the queue row, the data is then populated on the participant form, when taken to `/exisitingparticipant/` . <-- ill try and include a description of the 'results' of your feature, so you might catch stuff like this

_NOTE_
This is a  good time to mention that the way this is implemented is not the best, and leads to errors exactly like this. we are going to the store (the source of truth) and assigning new data directly to the row, then relying on the row data for some future action. 

A better approach would be to only get the data we need to render the row on the outset. When we have a further action that requires more data, use the participant _d and go back to the store for that data. We will fix this if we have time, I just want to point it out so you don't take everything in this repository as gospel

**Comment by MikeyManoguerra** [comment]
#407

File: frontend/src/constants/index.js (hunk 4 above)
Comment (dmjohnspor) [reply frontend/src/constants/index.js]: Done

**Comment by dmjohnspor** [comment]
Addressed all the issues

**Review by MikeyManoguerra — APPROVED** [review]

**Pull request merged**